app/forms/idv/doc_pii_form.rb

```diff
@@ -82,12 +88,6 @@ def dob_valid?
       end
     end
 
-    def zipcode_valid?
-      return if zipcode.is_a?(String) && zipcode.present?
-
-      errors.add(:zipcode, generic_error, type: :zipcode)
```

Comment:
This appears to have been the only reference to the `generic_error` method, so we probably ought to remove the method definition as being dead code.

Reply:
@aduth Good point, I made [a follow-up ticket](https://cm-jira.usa.gov/browse/LG-11954).